resources/integration_test/tests/dbmd_constant_result_sets.yml

```diff
@@ -32,94 +32,94 @@ tests:
     db: integration_test
     meta_function: [ getTypeInfo ]
     expected_sql_type: [ LONGVARCHAR, INTEGER, INTEGER, LONGVARCHAR, LONGVARCHAR, LONGVARCHAR,
-                         INTEGER, BOOLEAN, INTEGER, BOOLEAN, BOOLEAN, BOOLEAN, LONGVARCHAR, INTEGER, INTEGER, INTEGER,
-                         INTEGER, INTEGER ]
+                          INTEGER, BOOLEAN, INTEGER, BOOLEAN, BOOLEAN, BOOLEAN, LONGVARCHAR, INTEGER,
```

Comment:
Just curious, did you use a yaml formatting tool to correct this format?

Reply:
Ok thanks for letting me know.  No, we don't have another tool for the yaml so I was wondering how the formatting changed.  👍